tests/src/staking.test.ts

```diff
@@ -0,0 +1,561 @@
+import * as anchor from "@project-serum/anchor";
+import { Transaction } from "@raindrop-studios/sol-kit";
+import {
+  ItemProgram,
+  StakingProgram,
+  Instructions,
+  Utils,
+} from "@raindrops-protocol/raindrops";
+import { IDL as ItemProgramIDL } from "./types/item";
+import { IDL as StakingProgramIDL } from "./types/staking";
+import {
+  createMintNFTInstructions,
+  createMintTokensInstructions,
+} from "./utils/token";
+
+describe("Staking Program", () => {
+  it.concurrent("Item Staking", async () => {
+    const walletKeypair = anchor.web3.Keypair.generate();
+
+    const provider = new anchor.AnchorProvider(
+      new anchor.web3.Connection("http://localhost:8899"),
+      new anchor.Wallet(walletKeypair),
+      { commitment: "confirmed" }
+    );
+
+    await provider.connection.requestAirdrop(
+      walletKeypair.publicKey,
+      100 * anchor.web3.LAMPORTS_PER_SOL
+    );
+
+    const itemProgram = await ItemProgram.getProgramWithConfig(ItemProgram, {
+      asyncSigning: false,
+      provider,
+      idl: ItemProgramIDL,
```

Comment:
We have already exported these now to `js/lib/src/idls`, can use it like this: https://github.com/raindrops-protocol/raindrops/blob/master/rust/tests/namespace.ts#L451 

https://github.com/raindrops-protocol/raindrops/blob/master/rust/tests/namespace.ts#L11